src/api/player/server/getPlayerName.lua

```diff
@@ -0,0 +1,14 @@
+---getName
+---@param playerId number
+---@return string
+---@public
+function API_Player:getName(playerId)
+    if (playerId) then
+        local name = GetPlayerName(playerId)
+        if (name ~= nil) then
+            return (name)
+        end
+    end
+    _NCS:trace('Unable to find ID', 5)
+    return (false)
```

Comment:
```suggestion
    _NCS:die("Unable to find ID")
```
You can use `_NCS:die` since https://github.com/NextCitizens/ncs_core/commit/53bbdf825e1530fd24e0207bf7bdfcb1179f1a5c 😁

Reply:
Thx ! I change it now but i'm new sorry and how I change directly on this PR ?

Reply:
You just have to commit and to push from your fork !